### PR TITLE
support pythtest-conformance and pythtest-crosschain

### PR DESCRIPTION
**Pull request by cctdaniel**

### CODE REVIEW

File: pythclient/utils.py
Comment (cctdaniel) [inline pythclient/utils.py:1]: use hardcoded mapping values instead of DNS TXT record since these values dont change that often anyway

Comment (m30m) [reply pythclient/utils.py:1]: just curious, who is using these dns values?

Comment (cctdaniel) [reply pythclient/utils.py:1]: I actually dont know who updates these values, maybe @thmzlt?

**Review by m30m — APPROVED** [review]
please fix the comments before merge

**Pull request merged**